### ISSUE TITLE
Looks like there's a possible compatibility issue with Hugo

### ISSUE THREAD

**Issue by quintessence** [google/docsy#1333]
I was working on our docs site using the latest release of this theme and after testing a couple different versions of Hugo, it looks like the theme works as expected with `v0.101.0+extended` but not `v0.107.x+extended` and above. It basically doesn't seem to recognize the submodule at all in the more recent versions, but does with the earlier. I have a NAS so basically I have our code cloned there and I was pointing at it with two different machines, running the above versions of Hugo. The specific error in the later version is:

```
Error: from config: failed to resolve output format "print" from site config
```

Editing out the `print` from the `config.toml` file, e.g. from this:

```
# Comment out if you don't want the "print entire section" link enabled.
[outputs]
section = ["HTML", "print", "RSS"]
```

To this:

```
# Comment out if you don't want the "print entire section" link enabled.
[outputs]
section = ["HTML", "RSS"]
```

Does not resolve the error on the `v0.107.x` machine. The change is unnecessary on the `v0.101.0` machine :)

**Comment by LisaFC** [comment]
@gwatts could you take a look, I'm not sure what "print entire section" is doing that the latest version of Hugo doesn't like...

**Comment by raum51** [comment]
I have to say, that I wasn't able to reproduce my before discribed error. I tried to build it up from scretch, but **in my case** everything worked fine this time. Sorry!

**Comment by quintessence** [comment]
I noticed that this issue resolved with the `0.6.0` release (we were using `0.5.1` when I filed the bug report). Feel free to close whenever convenient :D

**Comment by deining** [comment]
Closing issue.
